src/profile-logic/process-profile.js

```diff
@@ -539,7 +650,11 @@ export function serializeProfile(profile) {
   return JSON.stringify(newProfile);
 }
 
-function unserializeProfile(profile) {
+/**
+ * Take a serialized processed profile from some saved source, and re-initialize
+ * any non-serializable classes.
```

Comment:
nit: add a line that it modifies the argument, with a TODO pointing to a new issue to change this.